### DISCUSSION TITLE
GPRS Shield V1.0 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1033] [Q&A]
# GPRS Shield V1.0 | Seeed Studio Wiki

GPRS Shield V1.0

https://wiki.seeedstudio.com/GPRS_Shield_V1.0

**Comment by Dmitriy-1986** [comment]
Hello World! ))

**Reply by MatthewJeffson** [reply]
Hey lo~
